vividus-soft-assert/src/test/java/org/vividus/softassert/model/SoftAssertionErrorTests.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.softassert.model;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.vividus.softassert.issue.KnownIssueType;
+
+public class SoftAssertionErrorTests
+{
+    private AssertionError assertionError = new AssertionError();
+    private SoftAssertionError softAssertionError = new SoftAssertionError(assertionError);
+
+    @Test
+    void testKnownIssueNull()
+    {
+        softAssertionError.setKnownIssue(null);
+        Assertions.assertFalse(softAssertionError.isKnownIssue());
+    }
+
+    @Test
+    void testKnownIssuePotentiallyKnownTrue()
+    {
+        KnownIssue knownIssue = new KnownIssue("Known issue", KnownIssueType.AUTOMATION, true);
+        softAssertionError.setKnownIssue(knownIssue);
+        Assertions.assertFalse(softAssertionError.isKnownIssue());
+    }
+
+    @Test
+    void testKnownIssuePotentiallyKnownFalse()
+    {
+        KnownIssue knownIssue = new KnownIssue("Not a known issue", KnownIssueType.AUTOMATION, false);
+        softAssertionError.setKnownIssue(knownIssue);
+        Assertions.assertTrue(softAssertionError.isKnownIssue());
```

Comment:
all this tests can be converted to a single parametrized one

Reply:
please check